server/channels/api4/websocket.go

```diff
@@ -48,6 +49,7 @@ func connectWebSocket(c *Context, w http.ResponseWriter, r *http.Request) {
 		TFunc:     c.AppContext.T,
 		Locale:    "",
 		Active:    true,
+		PostedAck: r.URL.Query().Get(postedAckParam) == "true",
```

Comment:
I don't have a better name for this, but I thought `PostedAck` was tracking whether or not an ack had been "posted" or sent over the connection. Turns out our possibly-too-concise message name leads to other names being potentially unclear

Reply:
Naming is hard.